tmol/score/omega/params.py

```diff
@@ -0,0 +1,62 @@
+import attr
+import cattr
+
+import pandas
+
+import torch
+import numpy
+
+
+from tmol.types.torch import Tensor
+from tmol.types.tensor import TensorGroup
+from tmol.types.array import NDArray
+from tmol.types.attrs import ValidateAttrs
+from tmol.types.functional import validate_args
+
+from tmol.database.scoring.omega import OmegaDatabase
+
+
+@attr.s(auto_attribs=True, slots=True, frozen=True)
+class OmegaGlobalParams(TensorGroup):
+    K: Tensor[torch.float32][...]
+
+
+@attr.s(frozen=True, slots=True, auto_attribs=True)
+class OmegaParamResolver(ValidateAttrs):
+    """Container for global/type/pair parameters, indexed by atom type name.
```

Comment:
FWIW, the "ParamResolver" idiom is mostly a holdover from the PackedResidueSystem way of approaching scoring. The ParamResolvers would take a set of residues and create denormalized tensors of parameters so that each atom had its own parameters and didn't need to be thought of as part of a particular residue. I don't believe you need a param resolver for omega; the tensor of the GlobalParams can be created simply by the OmegaEnergyTerm in its __init__ method.